src/ray/object_manager/test/pull_manager_test.cc

```diff
@@ -102,7 +102,16 @@ TEST_F(PullManagerTest, TestManyUpdates) {
     pull_manager_.OnLocationChange(obj1, client_ids, "");
   }
 
-  ASSERT_EQ(num_send_pull_request_calls_, 100);
+  // We shouldn't send multiple requests to the same node.
```

Comment:
I'm not sure if I understand the changes to the tests. I was expecting there to be some check that each client is only pulled from once.